package.json

```diff
@@ -65,7 +65,7 @@
     "tdd": "vitest watch",
     "e2e": "NEXT_PUBLIC_IS_E2E=1 yarn playwright test --project=@calcom/web",
     "e2e:app-store": "QUICK=true yarn playwright test --project=@calcom/app-store",
-    "e2e:embed": "QUICK=true yarn playwright test --project=@calcom/embed-core",
```

Comment:
QUICK was unused variable.

Adding E2E env variable to allow testing preveiw.html without iframe